### ISSUE TITLE
minikube: not running on latest version raises error

### ISSUE THREAD

**Issue by dprelipcean** [reanahub/reana-client#323]
As I'm not running the latest `minikube` version, I get the notification that there's a new version. However, when I try to set my environment variables, it actually fails. Funnily enough, when I run it a second time it goes through successfully.
```console
$ eval $(reana-dev setup-environment)
⚠️  minikube 1.3.1 is available! Download it: https://github.com/kubernetes/minikube/releases/tag/v/1.3.1
bash: export: `notice,': not a valid identifier
bash: export: `run:': not a valid identifier
bash: export: `minikube config set WantUpdateNotification false192.168.39.236:30443': not a valid identifier
$ eval $(reana-dev setup-environment)
````
Not sure whether we could do something about it more than suggest to use latest versions.

**Comment by tiborsimko** [comment]
Is it only due to the notification message?  What is the full output of:

```console
$ reana-dev setup-environment
```

i.e. without eval?

Note also that you can switch off the upgrade warning by doing:

```console
$ minikube config set WantUpdateNotification false
```

You may want to run it on your machine to silence these warnings and retry the command.

**Comment by diegodelemos** [comment]
This fails because [we do not do any check on the output of `minikube ip`](https://github.com/reanahub/reana-cluster/blob/e3155fa3495cafee5414ad19bc14b36d6afa0f70/reana_cluster/backends/kubernetes/k8s.py#L684-L686) in REANA-Cluster CLI, we always trust it will give an IP, which breaks setting the `REANA_SERVER_URL` environment variable when the output is something else (for example update warnings).

**Comment by tiborsimko** [comment]
Yeah, we could check `minikube ip` output to be ready for more situations, however setting `WantUpdateNotification` to false should also solve this. Since Minikube deployments are only used by developers, it should be sufficient, as people can run `update-check` manually...
